platforms/openid-connect/package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "unleash-docker",
```

Comment:
in order to get this automatically 🚢  to the dockerhub the folder needs to be added to the github action matrix.

`buildpush.yml` -> L:16
`buildtags.yml` -> L:13
`createtag.yml` -> L:13
`pr.yml` -> L:14

Then it will be automatically deployed when a new version of the unleash is merged into this repository.